### PR TITLE
build: Add install rule

### PR DESCRIPTION
**Pull request by rzr**
For debian packaging

Change-Id: I9953e111af7e94c76a825b4dbe5cdeec655175f5
Signed-off-by: Philippe Coval <p.coval@samsung.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -151,3 +164,7 @@ bluetoothdevice_image:
 .PHONY: bluetoothdevice_lint
 bluetoothdevice_lint:
 	make -C mappers/bluetooth_mapper lint
+
+install: ${exes}
+	install -d ${INSTALL_DIR}/bin/
+	install $^ ${INSTALL_DIR}/bin/
```
Comment (kadisi) [inline Makefile]: if we add `install` rule, We need to consider the installation path of the configuration file.

**Comment by rzr** [comment]
FYI this change is in latest snapshot binary:

https://dl.bintray.com/rzr/deb/pool/k/kubeedge/

/assign @qizha

**Comment by rzr** [comment]
I am waiting for this merge:
https://github.com/kubeedge/kubeedge/pull/1277

File: Makefile [hunk 2]
```diff
@@ -285,3 +289,11 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
+install-binaries: _output/local/bin
```
Comment (daixiang0) [inline Makefile:293]: I like it, would it better to name it just as `install`?
Also, `make install WHAT=keadm`  and so on to keep sync with `make all` could be better.

**Comment by rzr** [comment]
I just rebased on master branch

File: Makefile [hunk 3]
```diff
@@ -294,3 +298,13 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
```
Comment (chendave) [inline Makefile:301]: Can you please add some example here like build target? something like below, 
```
make install WHAT=edgecore
will install the binary to /usr/local/bin/ dir
```

**Review by chendave — COMMENTED** [review]
LGTM otherwise.

**Comment by chendave** [comment]
Thanks!

/lgtm
/approved

**Pull request merged**